pom.xml

```diff
@@ -301,7 +301,7 @@
                 <configuration>
                     <showWarnings>true</showWarnings>
                     <showDeprecation>true</showDeprecation>
-                    <compilerId>javac-with-errorprone</compilerId>
+                    <!-- <compilerId>javac-with-errorprone</compilerId> -->
```

Comment:
[Errorprone](http://errorprone.info/) is supposed to always run. Please revert this to the original state.